plaidml/edsl/tests/edsl_test.cc

```diff
@@ -1190,6 +1190,32 @@ TEST_F(CppEdsl, Cos) {
   checkProgram(program, {{S, A_input}}, {{O, C_output}});
 }
 
+TEST_F(CppEdsl, Sin) {
+  auto S = Placeholder(DType::FLOAT32, {3, 3});
+  TensorDim I, J;
+  TensorIndex i("i"), j("j");
+  S.bind_dims(I, J);
+  auto O = sin(S);
+  auto program = makeProgram("sin", {O});
+  std::cout << program << std::endl;
+  // clang-format off
+  // CHECK-LABEL: CppEdsl.Sin
+  // clang-format on
```

Comment:
Not sure what value we have with this if we aren't going to check the actual code.